### ISSUE TITLE
Configure ESLint to analyze ES6, not ES5?

### ISSUE THREAD

**Issue by Mark-L6n** [quasarframework/quasar-cli#47]
> By default, ESLint expects ECMAScript 5 syntax. You can override that setting to enable support for other ECMAScript versions …

([ESLint docs](https://eslint.org/docs/user-guide/configuring#specifying-parser-options))

Should Quasar configure ESLint for ECMAScript 6, 7 or 8?  This would be done by adding an `ecmaVersion` key to `.eslintrc.js`, for ex:

``` js
  parserOptions: {
    sourceType: 'module',
    ecmaVersion: 6
  },
```

OTOH, I don't know if this is working properly: I added this to handle [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), but still keep getting errors for them.

**Comment by JCharante** [comment]
fwiw I use template literals and didn't have to change my config.

**Comment by Mark-L6n** [comment]
Thanks @JCharante, I see that the template literals are working correctly.
My error was in thinking that the contents of the template literal wouldn't still be checked for errors.  For ex, I was getting lots of `Unexpected tab character` warnings for dummy tab-separated content I put in between backticks.
To solve that issue, I needed to put the backtick-delimited content inside `/* eslint-disable */ … /* eslint-enable */`
Since I stumbled across the `ecmaVersion` setting though, thought it would be good to point it out. ;-)
